test_plugins/SubstructuresPlugin.py

```diff
@@ -0,0 +1,88 @@
+from nanome.api.structure import molecule
+import nanome
+import sys
+import time
+
+# Config
+
+NAME = "Substructure Test Plugin"
+DESCRIPTION = "Simple plugin demonstrating the substructure features."
+CATEGORY = "Test"
+HAS_ADVANCED_OPTIONS = False
+
+# Plugin
+
+class SubstructuresPlugin(nanome.PluginInstance):
```

Comment:
For Test Plugins, can we add some `assert` calls, to verify the results that come back?